packages/perseus/src/widgets/__tests__/radio.test.ts

```diff
@@ -429,9 +426,6 @@ describe("single-choice question", () => {
 
             // Act
             await userEvent.keyboard(" ");
-            await act(async () => {
-                await jest.runAllTimers();
-            });
```

Comment:
@nicolecomputer I don't _think_ these `act` calls are necessary, because `userEvent` will advance the Jest timers for us. Do you agree? (asking because you added the act calls, but no worries if you have no memory of this)